PythonFiles/BlastBuilder.py

```diff
@@ -0,0 +1,32 @@
+from PythonFiles.Blast import Blast
+
+
+class BlastBuilder(object):
+
+    def set_WGS_paths(self, path_list: list):
+        self.WGS_paths = path_list
+
+    def set_WGS_flags(self, seq_type: str):
+        if seq_type is "N":
+            self.WGS_flags = "nucl"
+        else:
+            self.WGS_flags = "prot"
+
+    def set_query_paths(self, path_list: list):
+        self.query_paths = path_list
+
+    def set_query_flags(self, seq_type: str):
+        if seq_type is "N":
```

Comment:
We can extract a function here to combine the behavior here with the behavior of set_WGS_flags